src/locales/ko/dialogue.json

```
@@ -933,6 +1004,28 @@
       "1": "너희가 보기에는 내가 끔찍한 짓을 벌이고 있는 것처럼 보이겠지? 조금도 이해가 가지 않을 거야.\n$하지만 난 가라르지방의 영원한 번영을 위해서 무한한 에너지를 가져다줘야 해."
     }
   },
+  "star_boss_penny_1": {
+    "encounter": {
+      "1": "I'm the big boss of Team Star. The name's Cassiopeia. \n$Now, bow down before the overwhelming might of Team Star's founder!"
+    },
+    "victory": {
+      "1": "... ... .."
+    },
+    "defeat": {
+      "1": "Heh..."
+    }
+  },
+  "star_boss_penny_2": {
+    "encounter": {
+      "1": "I won't hold back in this battle! I'll stay true to Team Star's code! \n$My Veevee power will crush you into stardust!"
+    },
+    "victory": {
+      "1": "...It's all over now."
+    },
+    "defeat": {
+      "1": "I can't fault you on your battle skills at all... Considering how the bosses fell at your hands."
```

```suggestion
      "1": "내가 바로 스타단의 진 보스, 카시오페아… \n$…진 보스의 힘 앞에 무릎 꿇게 만들어 주겠어!!"
    },
    "victory": {
      "1": "… … …"
    },
    "defeat": {
      "1": "후후…"
    }
  },
  "star_boss_penny_2": {
    "encounter": {
      "1": "승부인 이상, 봐주지 않아! 그것이 스타단의 규칙이니까! \n$브이브이 파워로 우주의 먼지로 만들어 주겠어!!"
    },
    "victory": {
      "1": "이걸로 진짜 끝이구나…"
    },
    "defeat": {
      "1": "군더더기가 없는 실력이네. 보스들이 당한 걸 생각해 보면 말이야."
```
"Considering how the bosses fell at your hands." 부분의 원문은 "No wonder how the bosses fell at your hands."인데, 의미가 크게 달라진 것은 아니지만 미묘한 뉘앙스 차이가 있어 살려서 고쳐 보았습니다.